lms/djangoapps/courseware/access.py

```diff
@@ -132,9 +132,6 @@ def has_access(user, action, obj, course_key=None):
     if not user:
         user = AnonymousUser()
 
-    if isinstance(course_key, CCXLocator):
-        course_key = course_key.to_course_locator()
-
```

Comment:
@doctoryes in courseware app the masquerading code which we use for viewing courseware content as staff, student , specific student, it use same cache which it stores against course_key/ccx key. So when we use this line ```course_key = course_key.to_course_locator()``` it returns master course key instead of ccx key but masquerade code needs ccx (for viewing ccx courseware as staff, student or specific student)